### ISSUE TITLE
Explain flashing of FW in README

### ISSUE THREAD

**Issue by doniks** [tooyipjee/Spark-Analyzer#6]
Right now, documentation is quite sparse. It took me a while to figure out how to flash (https://github.com/tooyipjee/Spark-Analyzer/issues/2#issuecomment-2263427216) I think this could be better explained. I'd suggest a "Development" section in the README

**Comment by doniks** [comment]
basically:

* clone FW
* install PlatformIO
* test/configure serial port
* flash FW
* flash FS
* optionally, measure

**Comment by doniks** [comment]
maybe also explain buttons and (uart) pins?

**Comment by tooyipjee** [comment]
Could you make a PR for this explaination and feature?

**Comment by doniks** [comment]
Yes. Just want to wait for this https://github.com/tooyipjee/Spark-Analyzer/pull/3 to wrap up first

**Comment by doniks** [comment]
included in a commit here: https://github.com/tooyipjee/Spark-Analyzer/pull/10/commits
